README.md

```diff
@@ -332,21 +332,31 @@ export LLVM_SYS_180_PREFIX=/usr/lib/llvm-18
 export TABLEGEN_180_PREFIX=/usr/lib/llvm-18
 ```
 
-Run the deps target to install the other dependencies such as the cairo compiler (for tests, benchmarks).
```

Comment:
I removed it as it was already mentioned in the common setup (both macos and linux)